pkgs/development/tools/poetry2nix/poetry2nix/lib.nix

```diff
@@ -146,7 +146,7 @@ let
     else
       pkgs.fetchurl {
         url = predictURLFromPypi { inherit pname file hash kind; };
-        inherit hash;
+        sha256 = builtins.elemAt (builtins.match "sha256:(.*)" hash) 0; # nix 2.0 backwards compatibility.
```

Comment:
If we go this route, poetry2nix should probably be updated to handle this properly.

/cc @adisbladis